PROTOCOL.md

```diff
@@ -0,0 +1,75 @@
+# The XMTP Protocol
+
+## Intro
+
+The protocol allows `clients`, representing identities associated with blockchain accounts (further 'accounts'), to communicate securely with each other. The communications are represented as `conversations` between two or more clients. A conversation consists of a sequence of `messages` from a `sender` to the other `participants` of the conversation, the `recipients`.
+
+The communication is mediated by a network of `nodes`. The network is responsible for ensuring that messages are delivered between clients. Since clients are not necessarily connected to the network at the same time, the network must be able to store messages while they are in transit.
+
+The protocol is layered to satisfy various operational and security requirements. For example, the clients are responsible for all the encryption and authentication, and key management. The nodes generally see only encrypted content with no indication of identities involved.
```

Comment:
The word "generally" here makes me think the nodes might see unencrypted data. I know you explain it below, but I'm wondering if we can make this last sentence flex a little harder.

Maybe something like `The nodes see only encrypted user-generated content with no indication of identities involved`?

Reply:
I like that.